communications/newsletters/newsletter_14_May2020.md

```diff
@@ -0,0 +1,192 @@
+The Authors have been running a weekly reading group that can be attended online ([see details](http://datafeminism.io/blog/book/data-feminism-reading-group/)).
+
+## Past workshops and events
+
+### Women in Science - IBM Code Bristol
+
+Malvika Sharan was invited to the Women in Data Science (WiDS) event hosted by IBM Code Bristol to speak about _The Turing Way_ and its collaborative aspects.
+This event involved an all-women lineup of speakers who gave Tech Talks related to their work and participated in a panel discussion.
+You can watch the [recordings online](https://www.bigmarker.com/ibm-developer-uki/Women-in-Data-Science-IBM-Code-Bristol?bmid=2581093331c4).
+This was the first WiDs event at Bristol to coincide with the annual Global WiDS Conference held at Stanford University.
+WiDS initiative aims to inspire and educate data scientists worldwide, regardless of gender, and to support women in the field.
+
+### ReproHack 
+
+Anna Krystalli hosted an online ReproHack event on 14 May 2020, where Sarah Gibson delivered a talk on Binder.
+Both members are active contributors to _The Turing Way_ and fellows of the Software Sustainability Institute.
+[ReproHacks](https://github.com/reprohack) are one-day workshop events, where participants attempt to reproduce published research of their choice from a list of proposed papers with publicly available associated code and data.
+See [the workshop page](https://n8cir.org.uk/events/remote-reprohack/) for details. 
+
+## Tips & Tricks for new contributors
+
+Are you new to _The Turing Way_ and would like to learn more about the project? Listen to the insightful talk delivered by the project lead, Kirstie Whitaker, at [OpenMR Benelux 2020](https://openmrbenelux.github.io/products/product08/) recording from which is available [online on YouTube](https://www.youtube.com/watch?v=dlycvMU45ek&feature=youtu.be).
+
+### Contributor in focus: Camila Rangel Smith
+
+Camila Rangel Smith is a Research Data Scientist at The Alan Turing Institute.
+She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
+She is translating _The Turing Way_ resources into Spanish in collaboration with her friends and colleagues from Venezuela and Mexico. 
+In her words: *“I think The Turing Way is an excellent resource that can change the way science is done (I wish I had it when I started my PhD). 
+Although the international language of science is English, I know for a fact that not everyone in places like Latin-American has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone. 
```

Comment:
My fault again, I think is "Latin-America" without the "n" at the end.

Reply:
```suggestion
Although the international language of science is English, I know for a fact that not everyone in places like Latin-America has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone. 
```